examples/booking_app/lib/feature_hotel/search/views/search_page.dart

```diff
@@ -170,4 +145,198 @@ class _SearchPageState extends State<SearchPage> with TickerProviderStateMixin {
+                  return Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      FocusButton(
+                        onPressed: () {
+                          Alert(
+                            context: context,
+                            title: '',
+                            buttons: [],
+                            onWillPopActive: true,
+                            content: AdvancedFilterPage(
+                              roomCapacity: capacityData?.rooms ?? 0,
+                              personCapacity: capacityData?.persons ?? 0,
+                              onApplyPressed: (rooms, persons) {
+                                bloc.events.filterByCapacity(
+                                  roomCapacity: rooms,
+                                  personCapacity: persons,
+                                );
+                                Navigator.of(context).pop();
+                              },
+                            ),
+                          ).show();
+                        },
+                        child: Column(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: <Widget>[
+                            Text(
+                              'Capacity filters',
+                              style: TextStyle(
+                                  fontWeight: FontWeight.w100,
+                                  fontSize: 16,
+                                  color: Colors.grey.withOpacity(0.8)),
+                            ),
+                            const SizedBox(
+                              height: 8,
+                            ),
+                            Text(
+                              capacityFilterText,
+                              style: const TextStyle(
+                                fontWeight: FontWeight.w100,
+                                fontSize: 14,
+                              ),
+                            ),
+                          ],
+                        ),
+                      ),
+                      if (capacityFilterText != 'None')
+                        _buildClearButton(() {
+                          _presentYesNoDialog(
+                            context: context,
+                            title: 'Clear advanced filters?',
+                            onYesPressed: () {
+                              bloc.events.filterByCapacity(
+                                roomCapacity: 0,
+                                personCapacity: 0,
+                              );
+                            },
+                          );
+                        }),
+                    ],
+                  );
+                },
+              ),
+            ),
+          ],
+        ),
+      );
+
+  Future<void> _presentYesNoDialog({
```

Comment:
@DDavidPrime can we have this as a separate widget so we can use it from other places too ?